packages/gasket-plugin-nextjs/lib/create.js

```diff
@@ -0,0 +1,147 @@
+const { devDependencies } = require('../package');
+const { pluginIdentifier } = require('@gasket/resolve');
+
+/**
+ * createAppFiles
+ * @property {Files} files - The Gasket Files API.
+ * @property {generatorDir} - The directory of the generator.
+ */
+function createAppFiles({ files, generatorDir }) {
+  files.add(
+    `${generatorDir}/app/.*`,
+    `${generatorDir}/app/*`,
+    `${generatorDir}/app/**/*`
+  );
+}
+
+/**
+ * createTestFiles
+ * @property {Files} files - The Gasket Files API.
+ * @property {generatorDir} - The directory of the generator.
+ * @property {testPlugin} - Selected test plugin from prompt
+ */
+function createTestFiles({ files, generatorDir, testPlugin }) {
+  const frameworks = ['jest', 'mocha', 'cypress'];
+  frameworks.forEach((tester) => {
+    if (
+      testPlugin &&
+      pluginIdentifier(testPlugin).longName === `@gasket/plugin-${tester}`
```

Comment:
We need to move away from supporting short names and using `pluginIdentifier` so we can completely axe the `@gasket/register`. We'll need a different pattern here, but I'm fine if it's handled in a different PR.

Reply:
I overlooked this. With the removal of resolve this needs to change sooner rather than later. I'll update.